packages/system/Accounts/plugin/accounts/src/interfaces/api.rs

```diff
@@ -71,4 +72,9 @@ impl API for AccountsPlugin {
     fn get_current_user() -> Option<String> {
         AppsTable::new(&Client::get_active_app()).get_logged_in_user()
     }
+
+    fn gen_rand_account() -> Result<String, Error> {
```

Comment:
One minor improvement I just thought of is that this could take a `prefix` parameter, and the generated account will start with the prefix. So for example all "representative roles" could be prefixed with "r-" and all council roles with "c-", etc. Just to make the accounts very slightly more consistent/understandable without using a dedicated front-end